.github/workflows/codeql.yml

```diff
@@ -23,32 +23,43 @@ jobs:
       contents: read
       actions: read  # for github/codeql-action/init to get workflow details
       security-events: write  # for github/codeql-action/analyze to upload SARIF results
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - language: actions
+          - language: java
```

Comment:
Is having this explicit list what makes it slightly faster? To me, it's not completely clear how this PR improves performance. 🤷🏻